url/urlsearchparams-constructor.html

```diff
@@ -18,15 +18,25 @@
 }, 'Basic URLSearchParams construction');
 
 test(function() {
-    assert_throws(new TypeError(), function () { URLSearchParams(); },
-                  'Calling \'URLSearchParams\' without \'new\' should throw.');
-    assert_throws(new TypeError(), function () { new URLSearchParams(DOMException.prototype); });
-    var params = new URLSearchParams('');
+    var params = new URLSearchParams()
+    assert_equals(params.toString(), "")
+}, "URLSearchParams constructor, no arguments")
+
+test(() => {
+    params = new URLSearchParams(DOMException.prototype);
+    assert_equals(params.toString(), "Error=")
```

Comment:
https://github.com/w3c/web-platform-tests/pull/4581 has the fix. I also commented on the WebKit bug.